rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
should be able to remove this file now, right? We could pin it to 1.64.0 but I think that's not necessary now that any stable version is OK.

Reply:
Yes, this shouldn't be needed anymore.

Reply:
No wait, `nightly` is still needed for `#![feature(c_variadic)]`.